torch/csrc/jit/tensorexpr/llvm_codegen.cpp

```diff
@@ -519,7 +519,7 @@ void LLVMCodeGenImpl::emitKernel(
   // print graph debug info before optimization
   llvm::SmallVector<char, 0> asmBuffer;
   llvm::raw_svector_ostream asmStream(asmBuffer);
-  if (is_enabled(__FILE__, ::torch::jit::JitLoggingLevels::GRAPH_DEBUG)) {
+  if GRAPH_DEBUG_ENABLED {
```

Comment:
Clang format suggests to split `if` and `GRAPH_DEBUG_ENABLED {` in 2 lines. Below is the diff,
```
-  if GRAPH_DEBUG_ENABLED {
-    module_->print(asmStream, nullptr);
-  }
+  if
+    GRAPH_DEBUG_ENABLED {
+      module_->print(asmStream, nullptr);
+    }
```
Looks weird to me - add it or drop it?

Reply:
Maybe put it in parenthesis?